profiles/adb.jsonld

```diff
@@ -0,0 +1,231 @@
+{
+  "@context": "http://xapi.vocab.pub/profiles/profile-context.jsonld",
+  "id": "https://w3id.org/xapi/adb",
+  "type": "Profile",
+  "conformsTo": "https://github.com/DataInteroperability/xapi-profiles/tree/master#1.0-development",
+  "prefLabel": {
+  	"en": "Actionable Data Book (ADB) Profile"
+  },
+  "definition": {
+    "en": "The xAPI ABD profile is used to identify interactions between an actor and an eBook activity based on the IEEE ADB project."
+  },
+  "seeAlso": "https://standards.ieee.org/develop/indconn/adb/",
+  "author": {
+    "type": "Group",
+    "name": "IEEE Industry Connections Actionable Data Book (ADB) Team"
+  },
+  "versions": [
+    {
+      "id": "https://w3id.org/xapi/adb/v1",
+      "generatedAtTime": "2017-06-30T8:26:00Z",
+      "wasRevisionOf": ["https://w3id.org/xapi/adb"]
+    }
+  ],
+  "concepts": [
+    {
+      "id": "https://w3id.org/xapi/adb/verbs/annotated",
+      "inScheme": "https://w3id.org/xapi/adb",
+      "type": "Verb",
+      "definition": {
+        "en": "Provide notes or symbols for further explanation or thoughts while reading."
+      },
+      "narrower": [ {
```

Comment:
This is not the correct syntax for narrower. Narrower's value is an array of IRIs, not an array of objects.